src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const basicDiscount = 40;
+
+  if (days >= 7) {
+    return (basicDiscount * days - 50);
+  }
+
+  if (days >= 3) {
+    return (basicDiscount * days - 20);
+  }
+
+  return (basicDiscount * days);
```

Comment:
you repeat yourself, let's use one more variable to calculate total cost per days